lib/diff.js

```diff
@@ -87,7 +87,12 @@ var operations = {
     /*
      * { command: 'setTransition', args: [transition] }
      */
-    setTransition: 'setTransition'
+    setTransition: 'setTransition',
+
+    /*
+     * { command: 'setLighting', args: [lightProperties] }
+     */
+    setLighting: 'setLighting'
```

Comment:
@scothis I [changed it](https://github.com/mapbox/mapbox-gl-js/blob/building_redux_renderbuffer_dds/js/ui/light.js#L60-L72) while you weren't looking 🙈